panther_bringup/config/ekf_config.yaml

```diff
@@ -0,0 +1,75 @@
+# Ref: http://docs.ros.org/en/noetic/api/robot_localization/html/state_estimation_nodes.html
+
+frequency: 20
```

Comment:
Given that the lowest input frequency is 15Hz, is there a need to update the EKF at 20Hz? Do we want to rely on the IMU, despite its higher frequency, considering the higher interference it experiences?

Reply:
We KF always relay only current state and sensor, can be used to slightly increase the frequency of publication. I found 20 Hz to be a nice frequency.

Reply:
It could be also set to 20, but during our test we used 15 to tune other values, so I think that we should use 15 then.

Reply:
I test it also on 20 and looks fine